api/resource/definitions/runtime/runtime.proto

```diff
@@ -104,6 +104,21 @@ message MountStatusSpec {
   repeated string encryption_providers = 6;
 }
 
+// PCIRebindConfigSpec describes PCI rebind configuration.
+message PCIRebindConfigSpec {
+  string name = 1;
+  string vendor_device_id = 2;
```

Comment:
Is it just the model ID (like VID:PID)? Maybe there should be an option to select based on controller/port as well?